File: redwood-yoga-java/src/main/java/io/github/orioncraftmc/meditate/YogaNodeFactory.java
/*
 * Copyright (c) Facebook, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

package io.github.orioncraftmc.meditate;

public abstract class YogaNodeFactory {
  public static YogaNode create() {
    return new YogaNodeWrapper();
  }

  public static YogaNode create(YogaConfig config) {
    return new YogaNodeWrapper(config);
  }
}

File: redwood-yoga/src/commonMain/kotlin/app/cash/redwood/yoga/YogaNodeFactory.kt
/*
 * Copyright (c) Facebook, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
package app.cash.redwood.yoga

import kotlin.jvm.JvmStatic

object YogaNodeFactory {
  @JvmStatic
  fun create(): YogaNode {
    return YogaNodeWrapper()
  }

  @JvmStatic
  fun create(config: YogaConfig): YogaNode {
    return YogaNodeWrapper(config)
  }
}
